composer.json

```diff
@@ -27,11 +27,12 @@
         "yiisoft/strings": "^1.1"
     },
     "require-dev": {
-        "phpunit/phpunit": "^9.4",
-        "roave/infection-static-analysis-plugin": "^1.6",
+        "phpunit/phpunit": "^9.5",
+        "roave/infection-static-analysis-plugin": "^1.7",
         "spatie/phpunit-watcher": "^1.23",
-        "vimeo/psalm": "^4.2",
+        "vimeo/psalm": "^4.4",
         "yiisoft/aliases": "^1.1",
+        "yiisoft/active-record": "^3.0@dev",
```

Comment:
Why do you need AR to test DB?

Reply:
Because some methods are only tested in ActiveRecord, like ColumnSchema, `phpTypeCast()` and `insert()` in Schema::class, the best way to test them is with those who call those methods.

Reply:
I disagree. We should avoid using higher abstraction level packages for testing lower abstraction level packages.